15.js

```diff
@@ -0,0 +1,39 @@
+const readline = require('readline')
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+let state = 'CLOSED'
```

Comment:
Whenever I find myself using global state, I wonder if some entity (class) is missing in our domain...